### PR TITLE
Update to Python 3.

### PR DESCRIPTION
**Pull request by icemac**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -16,61 +16,72 @@
...
         'Operating System :: OS Independent',
+        'Programming Language :: Python :: 2',
+        'Programming Language :: Python :: 2.7',
+        'Programming Language :: Python :: 3',
+        'Programming Language :: Python :: 3.6',
+        'Programming Language :: Python :: 3.7',
+        'Programming Language :: Python :: Implementation :: CPython',
+        'Programming Language :: Python :: Implementation',
+        'Programming Language :: Python',
+        'Programming Language :: Python',
```
Comment (mgedmin) [inline setup.py]: These two last lines look unintentional.

**Review by mgedmin — APPROVED** [review]
LGTM, except for the weird classifiers.

**Comment by mgedmin** [comment]
I've activated [the repository on Travis](https://travis-ci.org/zopefoundation/z3c.preference).

**Comment by icemac** [comment]
> I've activated [the repository on Travis](https://travis-ci.org/zopefoundation/z3c.preference).

Interestingly the tests ran on travis-ci.com, see https://travis-ci.com/zopefoundation/z3c.preference

**Comment by mgedmin** [comment]
Oh, whoops!  I saw "build: unknown" on the badge and didn't realize it was because of course it was talking about git master.

I wonder if Travis will now do builds on both .org and .com.

**Comment by icemac** [comment]
@mgedmin wrote:
> I wonder if Travis will now do builds on both .org and .com.

Yes, it does. I am going to deactivate .org as .com seems to be the future.

**Pull request merged**